cypress/e2e/attributes/check-required-attributes.cy.js

```diff
@@ -16,22 +16,4 @@ describe('Testing disabling enabled attributes', () => {
       cy.gotoAttributePage(data.endpoint1, data.cluster1)
     })
   })
-  it(
```

Comment:
Why is this UI test removed?

Reply:
its checking for a client side attribute which means it is an incorrectly written test and should be remade in a separate PR

Reply:
Shouldn't it still be tested for protocols other than matter?